main.go

```diff
@@ -22,17 +19,22 @@ var (
 	checkName  = app.Flag("name", "check name").Default(appName).String()
 	chains     = app.Arg("chains", "iptables chains to monitor").Required().Strings()
 	ipVBin     = map[string]string{"v4": "iptables", "v6": "ip6tables"}
+	syslog     = app.Flag("syslog", "if set, logs stderr into syslog").Default("false").Bool()
 )
 
 func main() {
 	app.Version(version)
 	kingpin.MustParse(app.Parse(os.Args[1:]))
 
 	slog, err := syslog.New(syslog.LOG_NOTICE|syslog.LOG_DAEMON, appName)
+
```

Comment:
normally we leave the `if err ...` closer to the source, so this empty line can be deleted